### ISSUE TITLE
[BUG]: Installation error... widnows cannot find ....Minsudo.exe

### ISSUE THREAD

**Issue by ghost** [meetrevision/playbook#73]
### Describe the bug

```
Windows File not found 'C.\Program Files(x86)\Revision
Tool\flutter_assets\additionals\MinSudo.exe'
```

### Steps to reproduce

1. Install a fresh Windows 11 23H2 system from [MVS](https://massgrave.dev/genuine-installation-media.html)
Operating system version: 22631.3007
2. ReviOS Playbook 23.12
3. AME Wizard v0.7 Beta
4. Everything is installed by default

### Error logs is here
[AdminOutput.txt](https://github.com/meetrevision/playbook/files/13997971/AdminOutput.txt)
[ErrorLog.txt](https://github.com/meetrevision/playbook/files/13997972/ErrorLog.txt)
[Packages.txt](https://github.com/meetrevision/playbook/files/13997973/Packages.txt)
[TIOutput.txt](https://github.com/meetrevision/playbook/files/13997974/TIOutput.txt)

### Expected behavior

 I tried searching for issues with `MinSudo.exe` but couldn't find any information.

### Environment

- OS Build Number & Edition: `22631.3007`
- ReviOS Version: `23.12`
- Specs: `23H2`

### Have you searched for similar issues?

- [X] I have searched for similar issues and found none.

**Comment by boris1993** [comment]
Temporarily fixed by downloading the `minsudo` from [https://github.com/M2Team/NanaRun/releases](https://github.com/M2Team/NanaRun/releases), and putting it there. 
But still, "Completed with errors".

**Comment by melo936** [comment]
The problem is that Revision Tool hasn't installed for you which is strange. However, MinSudo or any other tool for permissions (such as TrustedInstaller) isn't required for the playbook anyway, as AME Wizard already supports using different levels of permissions.

**Comment by Lyceris-chan** [comment]
I performed a standard clean installation using ventoy as well as a ventoy-configured clean installation, and both processes ran smoothly without any issues. I executed the latest playbook based on the main branch on a Windows 11 ISO, which I created using UUPDump following the last Patch Tuesday release 22631.3007. Both installations were conducted without an internet connection, and I only connected to the internet after the ame wizard prompted for it. Everything is functioning correctly on my system. However, I did observe new installations of Spotify and Outlook, which may be attributed to queued store updates. Resetting the Microsoft Store and terminating it during the debloating process might resolve this issue.
